README.md

```diff
@@ -41,6 +44,11 @@ the `pip_virtualenv_command` variable.
 [venv]: http://virtualenv.readthedocs.org/
 [pip]: http://pip.readthedocs.org/
 
+Breaking Changes
+----------------
+
+As of 1.0.0, installation with git+pip, and from source, are no longer supported.
```

Comment:
> installation with git+pip ... no longer supported

I still need this. But I think it should be trivial to support if the argument to the `pip` module's `name` and `version` options are parameterized?

Reply:
Pushed bfd8883f9ff953bc231824431ac66b1d9cd7baf4 which should help

Reply:
```suggestion
As of 1.0.0, installation directly from source using `git clone` is no longer supported. However, `pulsar_package_name` and `pulsar_package_version` can be used to install using pip's `git+https` method.
```